AccountService.js

```diff
@@ -4,7 +4,7 @@
 'use strict';
 
 import axios from 'axios';
-
+const headers = {Accept: 'application/ld+json, application/json'};
```

Comment:
I still don't know if we need to override the default headers.  @gannan08 @dlongley

Reply:
good point we should seek a professional opinion also Content-Type should contain application/json-patch+json on the update method.

Reply:
Well, we do want to accept `application/ld+json` so it seems right to me. Why wouldn't it be?

Reply:
I tested this this morning and the default content type if application/json
we are not returning linked data yet, but not sure if we are in the future.
so @dlongley should I remove the application/ld+json Accept type or keep it because accounts might be linked data in the future?